ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -78,3 +82,60 @@ def pytest_sessionfinish(session, exitstatus):
         save_reports()
     if ocsci_config.RUN["cli_params"].get("email"):
         email_reports(session)
+
+    # creating report of test cases with total time in ascending order
+    data = GV.TIMEREPORT_DICT
+    sorted_data = dict(
+        sorted(data.items(), key=lambda item: item[1]["total"], reverse=True)
+    )
+    try:
+        time_report_file = os.path.join(
+            ocsci_log_path(), "session_test_time_report_file.csv"
+        )
+        with open(time_report_file, "a") as fil:
+            fil.write("testName\tsetup\tcall\tteardown\ttotal\n")
+            for test, values in sorted_data.items():
+                row = (
+                    f"{test}\t"
+                    f"{values.get('setup', 'NA')}\t"
+                    f"{values.get('call', 'NA')}\t"
+                    f"{values.get('teardown', 'NA')}\t"
+                    f"{values.get('total', 'NA')}\n"
+                )
+                fil.write(row)
```

Comment:
I think that it might be better to use [`csv`](https://docs.python.org/3/library/csv.html) module rather than creating the file "manually".